d3rlpy/base.py

```diff
@@ -437,7 +520,11 @@ def fit(
                 with logger.measure_time("step"):
                     # pick transitions
                     with logger.measure_time("sample_batch"):
-                        batch = next(iterator)
+                        # PEP 479
```

Comment:
Can you remove this comment and `try-catch` since this seems not to be relevant to this PR?

Reply:
ok !

Reply:
I think pylint will raise "stop-iteration-return" this is a known false positive of pylint

Reply:
I will add "stop-iteration-return" to the pylint disable list then